packages/select/src/common/listItemsProps.ts

```diff
@@ -12,12 +12,21 @@ import { ItemListPredicate, ItemPredicate } from "./predicate";
 /** Reusable generic props for a component that operates on a filterable, selectable list of `items`. */
 export interface IListItemsProps<T> extends IProps {
     /**
-     * The currently focused item for keyboard interactions, or `null` to
+     * The currently focused item for keyboard interactions, or `undefined` to
      * indicate that no item is active. If omitted, this prop will be
      * uncontrolled (managed by the component's state). Use `onActiveItemChange`
      * to listen for updates.
      */
-    activeItem?: T | null;
+    activeItem?: T;
+
+    /**
+     * Equality test implementation to determine if two values are representative of the same item.
+     * If not specified, then simple strict equality is used by default.
+     * @param valueA - A value.
+     * @param valueB - Another value.
+     * @return true if the two values are equivalent.
```

Comment:
you can remove these jsdoc tags as they don't provide additional information that isn't already in the main comment (and we're not in the habit of obsessively jsdoc-ing all arguments).

Reply:
Done